lib/phoenix_live_view/test/live_view_test.ex

```diff
@@ -1635,13 +1635,13 @@ defmodule Phoenix.LiveViewTest do
         }
       ])
 
-      assert render_upload(avatar, "foo.jpeg") =~ "100%"
+      assert render_upload(avatar, "myfile.jpeg") =~ "100%"
 
   By default, the entire file is chunked to the server, but an optional
   percentage to chunk can be passed to test chunk-by-chunk uploads:
 
-      assert render_upload(avatar, "foo.jpeg", 49) =~ "49%"
-      assert render_upload(avatar, "foo.jpeg", 51) =~ "100%"
+      assert render_upload(avatar, "myfile.jpeg", 49) =~ "49%"
+      assert render_upload(avatar, "myfile.jpeg", 51) =~ "51%"
```

Comment:
I believe this last example is incorrect– with `render_upload/3` progress is additive, so two consecutive invocations, with progress values of `49` and `51` respectively, should result in 100% progress:

```elixir
assert render_upload(avatar, "myfile.jpeg", 49) =~ "49%"
assert render_upload(avatar, "myfile.jpeg", 51) =~ "100%"
```

Reply:
Fixed.

Reply:
Thanks @mcrumm for explaining that, I had a test failing with a value above 100 😄